### PR TITLE
Add machine pool if the machine pool not exists with given name

### PR DESCRIPTION
**Pull request by aloganat**
Signed-off-by: Arthy Loganathan <aloganat@redhat.com>

### CODE REVIEW

File: utils/scripts/ocm/ocm.py [hunk 1]
```diff
@@ -49,6 +49,7 @@ def __init__(self, args={}):
         self.pool_node_count = args.get("pool_node_count")
         self.taints = args.get("taints")
         self.pool_name = args.get("pool_name")
+        self.reuse_machine_pool = args.get("reuse_machine_pool")
```
Comment (jgarciao) [inline utils/scripts/ocm/ocm.py:52]: This variable doesn't seem to be used

**Review by milindw96 — DISMISSED** [review]
Lgtm

File: utils/scripts/ocm/ocm.py (hunk 1 above)
Comment (aloganat) [reply utils/scripts/ocm/ocm.py:52]: Comments Incorporated.

**Review by milindw96 — APPROVED** [review]

File: utils/scripts/ocm/ocm.py (hunk 1 above)
Comment (jgarciao) [reply utils/scripts/ocm/ocm.py:52]: >       if bool(self.reuse_machine_pool) and self.check_if_machine_pool_exists():
Sorry, I didn't see this line before

**Review by jgarciao — APPROVED** [review]
lgtm

**Pull request merged**